cfy_manager/components/postgresql_client/postgresql_client.py

```diff
@@ -122,6 +126,13 @@ def configure(self):
     def remove(self):
         logger.notice('Removing PostgreSQL...')
         files.remove_notice(POSTGRESQL_CLIENT)
-        yum_remove('postgresql95')
-        yum_remove('postgresql95-libs')
+        rph = RpmPackageHandler(
+            'postgresql95-server-9.5.3-2PGDG.rhel7.x86_64.rpm')
+        if not rph.is_rpm_installed():
+            yum_remove('postgresql95')
+            yum_remove('postgresql95-libs')
+        else:
+            logger.notice('PostgreSQL Server installed on machine,'
+                          'will wait for other components to be removed prior'
+                          'to removing PostgreSQL')
         logger.notice('PostgreSQL successfully removed')
```

Comment:
How is PG successfully uninstalled? Nothing happend

Reply:
Fixed